hal/src/mrc/include/mrc/NetComm.h

```diff
@@ -178,6 +178,12 @@ struct MatchInfo {
                               EventName.size()};
   }
 
+  bool operator==(const MatchInfo& Other) const {
+    return MatchNumber == Other.MatchNumber &&
+           ReplayNumber == Other.ReplayNumber && Type == Other.Type &&
+           EventName == Other.EventName;
+  }
```

Comment:
```suggestion
  bool operator==(const MatchInfo& Other) const = default;
```
This should be exactly equivalent and more maintainable- See https://en.cppreference.com/w/cpp/language/default_comparisons for details.

Reply:
I'll make sure to do that during the refactoring.